### DISCUSSION TITLE
WebGIS6 - Desktop-Layout - Identifizieren

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#187] [Q&A]
Hallo Jürgen!

Im Desktop-Layout im WebGIS6 (6.24.2802) klicke ich in die Karte um Objekte zu identifizieren. Es geht das Fenster auf mit den Kategorien "Sichtbare Themen" und "Nicht sichtbare Themen". 

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/e2f39b3a-57b6-4f18-b990-c8a2d8fc0234)

Ich klicke auf den Text **"Nicht sichtbare Themen"** und es erscheint ein Fenster mit der Überschrift "Fehler" und der Meldung "Keine Abfragen gefunden!?". 

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/56685286-f96e-4c7d-b53b-d9aee35229d5)

Nach dem Wegklicken des Fensters erscheint unter "Nicht sichtbare Themen" ein sich dauernd drehendes Icon und daneben die Info "Abfragen...". 

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/ec985abd-9670-4635-9398-c95e6ef6eaaf)

Was mich irritiert. Es gibt keine "Nicht sichtbaren Themen", es wurden keine weiteren Abfragen konfiguriert. Als User bekomme ich allerdings ein Fenster mit einer Fehlermeldung und im Auswahlfenster hat es dann für mich den Anschein als würde das WebGIS versuchen "Nicht sichtbare Themen" zu laden, aber es funktioniert nicht (rotierendes Icon). 

Wäre es sinnvoll/möglich im Falle keiner "Nicht sichtbaren Themen" einen Hinweis zu geben, dass es keine Abfragen gibt oder überhaupt keine Meldung auszugeben und bei "Nicht sichtbare Themen" einfach nichts anzuzeigen bzw. ebenfalls den Hinweis "Keine Abfrageergebnisse gefunden" (analog zu "Sichtbare Themen").
 
![image](https://github.com/gis-eni/webgis-forum/assets/122986367/27d82c39-d52d-4923-b1e7-2a51c08099d2)

Ich hoffe es ist verständlich beschrieben. Bei Fragen bitte melden!

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Ok, muss erst einmal schauen, ob ich das nachvollziehen kann. Wir haben das Verhalten so bisher nicht.

**Reply by gis-eni** [reply]
Ich kann es leider nicht nachvollziehen :(
Bei mir schaut das immer so aus:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/4e8d5747-6b5a-45fc-a2f8-fa4855224ff8)

Und so ist es auch gewollt.

Wenn schon eine Karte gibt, die ich mir anschauen kann, schick mir einfach einen Link (per Mail)

**Reply by gis-eni** [reply]
Kommando zurück. Hab einen Weg gefunden, dass nachzuvollziehen. Lösung kommt...

**Reply by gis-eni** [reply]
Neuer Build liegt zum Download bereit: 6.24.2803

**Comment by Andreas-Lampert** [comment]
Vielen Dank Jürgen! Funktioniert super!
